images/openeuler.yaml

```diff
@@ -4,7 +4,7 @@ image:
     openEuler {{ image.release }}
 source:
   downloader: openeuler-http
-  url: https://mirrors.aliyun.com/openeuler/
+  url: http://mirrors.ocf.berkeley.edu/openeuler
```

Comment:
That Berkeley mirror seems to support HTTPS too.

Reply:
Done